qemu/tests/cfg/data_plane_snapshot.cfg

```diff
@@ -0,0 +1,13 @@
+- data_plane_snapshot:
```

Comment:
could you create a data_plane.cfg, and snapshot as a variant, instead of a separate cfg file,
or there will be lots of data plane related cfg file.